packages/build/src/download-center.ts

```diff
@@ -67,11 +67,17 @@ const CONFIG = `
           "download_link": "https://downloads.mongodb.com/compass/mongosh-{{version}}-linux.tgz"
         },
         {
-          "arch": "x64",
+          "arch": "amd64",
```

Comment:
We build debian package for `amd64`, so I think this should also be specified as `amd64`.

Reply:
oh nvm this.